packages/code-du-travail-frontend/src/common/ServiceRenseignement.js

```diff
@@ -61,28 +65,30 @@ function ServiceRenseignement() {
           Service gratuit <span>+ prix appel</span>
         </Pricing>
       </NumberInsert>
-
       <Heading as="h4">Contact par email et prise de rendez-vous</Heading>
-      <p>
-        <UID name={(id) => `id_${id}`}>
-          {(id) => (
-            <>
-              <label htmlFor={id}>
-                Saisissez votre numéro de département&nbsp;:
-              </label>
-              <StyledInput
-                id={id}
-                type="text"
-                name="search-service"
-                autoComplete="off"
-                maxLength="3"
-                placeholder="ex. 31, 35, 75"
-                onChange={onDepartmentInput}
-              />
-            </>
-          )}
-        </UID>
-      </p>
+      <Form
+        onSubmit={onDepartmentInput}
+        render={({ handleSubmit }) => (
+          <StyledForm onSubmit={handleSubmit}>
+            <label htmlFor="search-service">
+              Saisissez votre numéro de département&nbsp;:
+            </label>
+            <Field
```

Comment:
The field should have an id that match the htmlfor value of the label

Reply:
My bad, I see after that you put it on the render prop.imho It would be better to move to the field

Reply:
Yes, good idea !